src/WorldClock.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { Form, Button } from "react-bootstrap";
+
+class WorldClock extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      display: <></>,
+    };
+    this.timezonesInput = React.createRef();
+    this.inputForm = (
+      <Form className="w-50" onSubmit={() => this.submit()}>
```

Comment:
Do we really need a form? I think a simple input + state update would be sufficient here. Or do you have a reason to use a form?